src/dotnet/APIView/APIView/FieldAPIV.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.CodeAnalysis;
+using System.Text;
+
+namespace APIView
+{
+    /// <summary>
+    /// Class representing a field in a C# class/interface.
+    /// 
+    /// Field is an immutable, thread-safe type.
+    /// </summary>
+    public class FieldAPIV
+    {
+        public string Name { get; }
+        public string Type { get; }
+
+        public bool IsConstant { get; }
+        public bool IsReadOnly { get; }
+        public bool IsStatic { get; }
+        public bool IsVolatile { get; }
+
+        public object Value { get; }
+
+        /// <summary>
+        /// Construct a new Field instance, represented by the provided symbol.
+        /// </summary>
+        /// <param name="symbol">The symbol representing the field.</param>
+        public FieldAPIV(IFieldSymbol symbol)
+        {
+            this.Name = symbol.Name;
+            this.Type = symbol.Type.ToDisplayString();
+
+            this.IsConstant = symbol.HasConstantValue;
+            this.IsReadOnly = symbol.IsReadOnly;
+            this.IsStatic = symbol.IsStatic;
+            this.IsVolatile = symbol.IsVolatile;
+
+            if (symbol.HasConstantValue)
+                this.Value = symbol.ConstantValue;
+        }
+
+        public override string ToString()
```

Comment:
Instead of having a second ToString implementation in every model type make Renderer methods internal and call into them to get the string representation.